src/ServiceManagement/Compute/Commands.ServiceManagement/Extensions/Common/BaseAzureServiceExtensionCmdlet.cs

```diff
@@ -285,6 +288,15 @@ protected DeploymentGetResponse GetDeployment(string slot)
             return d;
         }
 
+        protected DeploymentGetResponse GetPeerDeployment(string currentSlot)
+        {
+            var currentSlotType = (DeploymentSlot)Enum.Parse(typeof(DeploymentSlot), currentSlot, true);
+            var peerSlot = currentSlotType == DeploymentSlot.Production ? DeploymentSlot.Staging : DeploymentSlot.Production;
+            var peerSlotStr = peerSlot.ToString();
```

Comment:
NIT: peerSlotStr is redundant.   Just call GetDeployment with peerSlot.ToString().